.github/workflows/linting.yml

```
@@ -0,0 +1,43 @@
+name: Linting
+
+on:
+  push:
+    branches: [ development ]
+  pull_request:
+    branches: [ development ]
+
+  
+jobs:
+  make-documentation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+    
+      - name: Install dependencies
+        run: sudo apt install libfftw3-dev libtiff5-dev libboost-all-dev libeigen3-dev libyaml-cpp-dev libcfitsio-dev casacore-dev libllvm7 
+    
+      - name: before script #extra dependencies? trusty?
+        run: |
+          curl https://apt.llvm.org/llvm-snapshot.gpg.key | sudo apt-key add -
+          echo "deb http://apt.llvm.org/trusty/ llvm-toolchain-trusty-7 main" | sudo tee -a /etc/apt/sources.list
```

ubuntu-latest is 20.04 [according to GH's vritual envs](https://github.com/actions/virtual-environments#available-environments)
```suggestion
          echo "deb http://apt.llvm.org/focal/ llvm-toolchain-focal main" | sudo tee -a /etc/apt/sources.list
```
you can find llvm details in [their apt page](https://apt.llvm.org/)

---

This is the linting action which should have been on a separate branch, I've made the requested changes on the correct branch and removed this file from this branch.